pkg/controller/jaeger/elasticsearch.go

```diff
@@ -61,6 +63,14 @@ func waitForAvailableElastic(c client.Client, es esv1alpha1.Elasticsearch) error
 	return wait.PollImmediate(time.Second, 2*time.Minute, func() (done bool, err error) {
 		depList := corev1.DeploymentList{}
 		if err = c.List(context.Background(), client.MatchingLabels(es.Labels).InNamespace(es.Namespace), &depList); err != nil {
+			if k8serrors.IsNotFound(err) {
```

Comment:
Did you run into this issue in general for any deployment? Interestingly I haven't see this for ES and the controller should reach this block before the deployment is found

Reply:
I did *not* run into this problem with ES, but in theory, it could also happen there. I guess it happens more often with deployments because we have two of them (query/collector)?